msprime/simulations.py

```diff
@@ -2178,6 +2178,95 @@ def population_size_trajectory(self, steps):
             N_t[j] = N
         return N_t
 
+    def possible_lineages(self, samples=None):
+        """
+        Given the sampling configuration, this function determines when lineages are
+        possibly found within each population over epochs defined by demographic events
+        and sampling times. If no sampling configuration is given, we assume we sample
+        lineages from each population at time zero. The samples are specified by a list
+        of msprime Sample objects, so that possible ancient samples may be accounted for.
+
+        :param list samples: A list of msprime Sample objects, which specify their
+            populations and times.
+        :return: Returns a list of times defining the epoch intervals and a list of
+            lists, one for each epoch, with lengths equal to the number of populations
+            and containing 1s or 0s for whether lineages could be found in that
+            population.
```

Comment:
It seems this function returns a dictionary of lists?

Reply:
You're right.. I had changed the output format and forgot to update the docstring. Will fix that.